src/components/button/submit-button.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import React from 'react'
+import { Button, createStyles } from '@mantine/core'
+
+const useStyles = createStyles((theme) => ({
+  button: {
+    color: theme.black,
+    backgroundColor: theme.colors.red[6],
+    '&:hover': {
+      color: theme.black,
+      backgroundColor: theme.colors.red[8],
+    },
+  },
+}))
+
+export function SubmitButton({ text }: { text: string }) {
```

Comment:
move `{ text: string }` to an interface called Props. There should be other examples of how this is done in other components.